OrdersApplication/Extensions/UIColorExtensions.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UIColorExtensions.swift
+//  OrdersApp
+//
+//  Created by Esraa Khaled   on 19/09/2022.
+//
+
+
+import UIKit
+
+extension UIColor {
+    static var brandPrimary = UIColor(named: "BrandPrimary")
+    static var borderPrimary = UIColor(named: "BorderPrimary")
```

Comment:
Since no change is intended so it should be constant. (Computed variable might be a good solution also)
```suggestion
    static let brandPrimary = UIColor(named: "BrandPrimary")
    static let borderPrimary = UIColor(named: "BorderPrimary")
```